cv32e40x/tests/programs/custom/pulp_bit_manipulation/pulp_bit_manipulation.S

```diff
@@ -1,762 +0,0 @@
-#
```

Comment:
I'm starting to wonder if maybe you are deleting these files.  GitHub is telling me that you are deleting all lines, but the file still exists.